rqt_bag_plugins/src/rqt_bag_plugins/plot_view.py

```diff
@@ -501,11 +501,27 @@ def _add_msg_object(self, parent, path, name, obj, obj_type):
 
             self._add_msg_object(item, subpath, subobj_name, subobj, subobj_type)
 
+    # Keyboard handler
+    def on_key_press(self, event):
+        key, ctrl = event.key(), event.modifiers() & Qt.ControlModifier
+        if ctrl:
+            if key == ord('C') or key == ord('c'):
+                # Ctrl-C: copy text from selected items to clipboard
+                self._copy_text_to_clipboard()
+                event.accept()
+            elif key == ord('A') or key == ord('a'):
+                # Ctrl-A: select all
+                self.expandAll()
+                self.selectAll()
+
     def handleChanged(self, item, column):
         if item.data(0, Qt.UserRole) == None:
             pass
         else:
-            path = self.get_item_path(item)
+            # Strip the leading underscore from each of the path segments
+            segments = [segment[1:] for segment in self.get_item_path(item).split('.')]
```

Comment:
Might be worth confirming that we're actually stripping a leading underscore, e.g.
```suggestion
            segments = [segment[1:] if segment[0] == '_' else segment for segment in self.get_item_path(item).split('.')]
```